foundation/foundation-mda/src/main/resources/templates/notifications/notification.configuration.store.tilt.vm

```diff
@@ -0,0 +1,14 @@
+project_path = os.path.abspath('.')
+# Update configuration_files_path to match the path of the config files to be loaded into the configuration store. Example 'src/main/resources/config'
+configuration_files_path = ''
+# For WSL users, the configuration files need to be in an accessible path. Use this prefix to add the path to the root shared file store. Example: '/mnt/c' or '/mnt/wsl/rancher-desktop'
+configuration_files_path_prefix = ''
```

Comment:
A: Since we expect this to be used only for WSL, I might make the variable name indicate that. E.g. `wsl_path_prefix`